### PR TITLE
projects:ad777x_iio: Add support for AD777x

### PR DESCRIPTION
**Pull request by jansunil**
Release AD777x IIO FW

### CODE REVIEW

File: projects/ad777x_iio/README.txt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+-Products supported: AD7770, AD7771, AD7779
+
+Overview
+--------
+This is a IIO based firmware application to evalute the AD777x device.
+This code was developed and tested on SDP-K1 controller board for mbed platform : https://os.mbed.com/platforms/SDP_K1/.
+and the NucleoL552ZEQ Board for STM32 platform: https://os.mbed.com/platforms/ST-Nucleo-L552ZE-Q/
+
+Product details: 
+https://www.analog.com/en/products/ad77770.html
```
Comment (SaikiranGudla) [inline projects/ad777x_iio/README.txt:12]: typo: remove extra "7". Same with line 13, 14

Comment (jansunil) [reply projects/ad777x_iio/README.txt]: The typo has been corrected now

**Comment by jansunil** [comment]
Change log:
1. Removed typo with the product page links

File: projects/ad777x_iio/scripts/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+numpy==1.22.4
+pyadi-iio==0.0.12
```
Comment (SaikiranGudla) [inline projects/ad777x_iio/scripts/requirements.txt:2]: nit: Shall we update this to the latest versions like 15?

Comment (jansunil) [reply projects/ad777x_iio/scripts/requirements.txt]: Yes, the change has been done

**Comment by jansunil** [comment]
Changelog:
Updated version of pyadi-iio to 15 in the requirements.txt

**Review by SaikiranGudla — APPROVED** [review]

**Comment by jansunil** [comment]
@mphalke - Do these changes look good for release?

**Review by mphalke — APPROVED** [review]

**Comment by mphalke** [comment]
> @mphalke - Do these changes look good for release?

lgtm

**Pull request merged**